screens/Home/MyVcsTabController.ts

```diff
@@ -56,6 +57,10 @@ export function useMyVcsTab(props: HomeScreenTabProps) {
       settingsService,
       selectShowHardwareKeystoreNotExistsAlert,
     ),
+    isVcLoaded: useSelector(vcService, selectIsVcLoaded),
```

Comment:
can we renamed these variables from Loaded to downloaded?

loading can be from storage also.